index.html

```diff
@@ -766,7 +770,7 @@ <h2>3. We wish you good voting!</h2>
                     this.userData.municipality.email = result.selectedObject.email;
                 },
                 birthdateSelected: function(result) {
-                    // this.userData.birthdate = result;
+                    this.userData.birthdate = result;
```

Comment:
měl jsem nějaký konkrétní důvod, proč tohle bylo vykomentované, který mi teď uniká… nějaký důvod, proč to má být aktivní?

Reply:
máš pravdu, funguje to i bez toho